### PR TITLE
R Cheat Sheet: Fix Unescaped Brackets

### PR DESCRIPTION
**Pull request by laurenancona**
@GuiltyDolphin here's that patch

https://duck.co/ia/view/r_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by GuiltyDolphin** [comment]
Something weird going on in one of the fields:

screenshot from 2016-03-11 18 45 53

File: share/goodie/cheat_sheets/json/r.json [hunk 1]
```diff
@@ -94,31 +94,31 @@
...
     }, {
       "key": "tail(v, 1)",
       "val": "select last"
     }, {
-      "key": "mat[1,]",
+      "key": "mat\\[1,\\]",
       "val": "select row 1"
     }, {
-      "key": "mat[, 2])",
+      "key": "mat\\[,2\\])",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/r.json]: should there be an opening `(` here?

**Comment by moollaza** [comment]
@GuiltyDolphin they all look fixed to me now?

I should have caught this before, my bad :frowning:

**Comment by GuiltyDolphin** [comment]
@moollaza Hold on, we seem to have a line-wrapping issue.

**Comment by GuiltyDolphin** [comment]
@moollaza Ack, Dax doesn't seem to want to deploy! Mind deploying manually?

**Comment by GuiltyDolphin** [comment]
@moollaza @laurenancona Seems to look good locally (for all of us), I think Dax is just in a bad mood :dolphin:

**Comment by tagawa** [comment]
@laurenancona, thanks again for fixing the R cheat sheet! You’ve done a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Pull request merged**